src/executable-code/executable-fragment.js

```diff
@@ -422,6 +422,19 @@ export default class ExecutableFragment extends ExecutableCodeTemplate {
     // don't need to create additional editor options in readonly mode.
     if (readOnly) return;
 
+    let highlightWithImports = () => {
```

Comment:
why do you need highlightWithImports? AFAIK we decided to use just highlight for both cases

Reply:
Ok, I'll rename the method) It does what we agreed on